src/components/dashboard/Dashboard.js

```diff
@@ -787,15 +798,14 @@ const getStylesFromProps = ({ theme }) => ({
     marginRight: normalize(-20),
     alignSelf: 'stretch',
   },
-  bigNumberStyles: {
-    fontSize: 42,
+  bigNumberStyles: balance => ({
```

Comment:
no need to add calc functions here.
1. keep it as plain object, remove all calcucations
2. calculate extra style containing font weight in the component, wrap it onto useMemo
3. apply this style, then extra style calcularted in the component